File: Examples/ProgrammingWithDocuments/WorkingWithStyles/WorkingWithStylesExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammingWithDocuments.WorkingWithStyles
{
    [TestClass, TestFixture]
    public class WorkingWithStylesExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void ExtractContentBasedOnStyles()
        {
            TestHelper.SetDataDir("ProgrammingWithDocuments/WorkingWithStyles/ExtractContentBasedOnStyles");

            ExtractContentBasedOnStylesExample.Program.Main();
        }

    }
}

File: Examples/ProgrammingWithDocuments/WorkingWithStyles/WorkingWithStylesExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammingWithDocuments.WorkingWithStyles
	<TestClass, TestFixture> _
	Public Class WorkingWithStylesExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub ExtractContentBasedOnStyles()
	            TestHelper.SetDataDir("ProgrammingWithDocuments/WorkingWithStyles/ExtractContentBasedOnStyles")
	
	            ExtractContentBasedOnStylesExample.Program.Main()
        End Sub

	End Class
End Namespace